src/calculateRentalCost.js

```diff
@@ -28,21 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  let sum = 0;
+  const sum = days * 40;
+  let total = 0;
 
   if (days < 3) {
-    sum = days * 40;
+    total = sum;
   }
 
   if (days >= 3) {
```

Comment:
I think here you could add a restriction in the condition so that it is greater than 3, but less than 7